sql/6_0_0-to-6_1_0_upgrade.sql

```diff
@@ -199,3 +199,30 @@ ALTER TABLE `procedure_providers` ADD `type` VARCHAR(31) DEFAULT NULL;
 ALTER TABLE `procedure_answers` ADD `procedure_code` VARCHAR(31) DEFAULT NULL;
 #EndIf
 
+#IfNotRow users username oe-system
+INSERT INTO `users`(`username`,`password`,`lname`,`authorized`,`active`) VALUES ('oe-system','NoLogin','System Operation User',0,0);
+INSERT INTO `gacl_aro`(`id`, `section_value`, `value`, `order_value`, `name`, `hidden`)
+    SELECT max(`id`)+1,'users','oe-system',10,'System Operation User', 0 FROM `gacl_aro`;
+INSERT INTO `gacl_groups_aro_map`(`group_id`, `aro_id`)
+    VALUES (
+        (SELECT `id` FROM `gacl_aro_groups` WHERE parent_id=10 AND value='admin')
+        ,(SELECT `id` FROM `gacl_aro` WHERE `section_value` = 'users' AND `value` = 'oe-system')
+    );
+#EndIf
+
+#IfNotTable
+CREATE TABLE `export_job` (
+  `uuid` varchar(40) NOT NULL,
+  `user_id` varchar(40) NOT NULL,
+  `client_id` varchar(80) NOT NULL,
+  `status` varchar(40) NOT NULL,
+  `start_time` datetime NOT NULL,
+  `resource_include_time` datetime DEFAULT NULL,
+  `output_format` varchar(128) NOT NULL,
+  `request_uri` varchar(128) NOT NULL,
+  `resources` text NOT NULL,
```

Comment:
make text bare:
```
`resources` text,
```

Reply:
This is also a required field, not sure why we would make it bare?

Reply:
old mariadb/mysql versions do not support setting of default for text (it just defaults to NULL). There was a large project in past that involved converting these all over to be bare (ie. just default to NULL), so am just keeping this standard going. Since it is a required field (like the date above) these changes should not impact use (since they will never be empty).

Reply:
TODO for me. revisit/review why recommending bare text

Reply:
@adunsulag , see the above screenshot for date where I looked at what happens when don't insert anything in a text bare and text not null field. On initial evaluation, this seems ok. However, the problem here is that when users upgrade to 5.0.0, there is code that will basically convert this to bare text (ie. default to null).
https://github.com/openemr/openemr/blob/master/sql/4_2_2-to-5_0_0_upgrade.sql#L89-L92

back in 2015 or so:
https://community.open-emr.org/t/error-when-creating-tables-with-text-columns-with-default-values/7460

fix in 2016:
https://github.com/openemr/openemr/commit/c13ddcbdb71919e3a8223c00c6a28ae3e4c0221d

Because of this, seems like makes most sense to just stick with the pattern of bare TEXT (ie. some users will likely run upgrade inadvertently from older versions and make it that ways anyways :) ) unless there is a big advantage by using NOT NULL.